docs/1_usage.md

```diff
@@ -2,69 +2,53 @@
 
 Lean how to use the ReVanced CLI.
 
-## ⚡ Setup (optional)
+## ⚡ Setup
 
-1. Make sure your device is connected
+1. Download Java 19: https://jdk.java.net/19/ then unzip it
+2. Download latest release (.jar) of revanced CLI: https://github.com/revanced/revanced-cli/releases
+3. Download latest release (.jar) of revanced patches: https://github.com/revanced/revanced-patches/releases
+4. Download latest release (.apk) of revanced integrations: https://github.com/revanced/revanced-integrations/releases
+5. Download the appropriate version of the Youtube APK: https://www.apkmirror.com/apk/google-inc/youtube/youtube-18-03-36-release/ (replace the version in the URL if necessary; make sure to download the APK and not the APK bundle)
```

Comment:
This setup is not correct. The CLI has nothing to do with YouTube, yet it is a required setup. Additionally, you limit the setup to JDK 19 and ApkMirror, whereas ReVanced has nothing to do with ApkMirror. The original setup concerned ADB, which is why the setup [was optional](https://github.com/revanced/revanced-cli/pull/203/files?diff=split&w=0#diff-be63be9ab2321a89ccbfad46365f58d7ea3e68ba660d7a7bfe5d53259975d1fcL5).